elasticdl/doc/tutorials/elasticdl_local.md

```diff
@@ -0,0 +1,112 @@
+# ElasticDL on local
+
+This document aims to give a simple example to show how to sumbmit deep learning jobs to a local kubernetes cluster in a local computer. It helps to understand the working process of ElasticDL.
+
+
+## Environment prepare
+
+Here, we use macbook as our expriment environment.
```

Comment:
We don't need to mention this if we just put a reference to minikube installation guide.

Reply:
Done